utils/cryptodev-linux/Makefile

```diff
@@ -10,12 +10,15 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=cryptodev-linux
-PKG_VERSION:=1.8
-PKG_RELEASE:=2
+PKG_VERSION:=2017-02-09
```

Comment:
@hnyman @nmav don't know... if it good the way hnyman suggested or putting the date in the PKG_RELEASE 
What do you think? opkg should upgrade it anyway... 
(anyway can't underestand the problem with opkg... it's a kernel package... it can't be upgraded anyway so... it shouldn't make a difference if we mantain the old version or we switch to the date thing)